### DISCUSSION TITLE
Can GitHub Importer import Subversion?

### DISCUSSION THREAD

**Discussion by Rod-at-DOH** [github/gh-gei#1253] [Q&A] [answered]
I'm going through one of the GitHub Skills trainings on GitHub Importer. It said that it can import Subversion repositories. But when I looked at the README, it didn't say anything about Subversion.

So, can GitHub Importer be used to import a Subversion repository?

**Comment by boylejj** [comment, marked as answer]
Hi @Rod-at-DOH. This functionality has been retired. The announcement can be found here: https://github.blog/changelog/2024-04-17-updates-to-github-importer-and-the-deprecation-of-the-source-import-rest-api-endpoint/

**Reply by Rod-at-DOH** [reply]
OK, thanks.
